omnibus/config/software/jmxfetch.rb

```diff
@@ -26,6 +26,22 @@
 build do
   ship_license "https://raw.githubusercontent.com/DataDog/jmxfetch/master/LICENSE"
   mkdir jar_dir
+
+  if osx?
+    # Also sign binaries and libraries inside the .jar, because they're detected by the Apple notarization service.
+    command "unzip jmxfetch.jar -d ."
+    delete "jmxfetch.jar"
+
+    if ENV['HARDENED_RUNTIME_MAC'] == 'true'
+      hardened_runtime = '-o runtime '
+    else
+      hardened_runtime = ''
+    end
+
+    command "find . -type f | grep -E '(\\.so|\\.dylib|\\.jnilib)' | xargs codesign #{hardened_runtime}--force --timestamp --deep -s 'Developer ID Application: Datadog, Inc. (JKFCB4CN7C)'"
+    command "zip jmxfetch.jar -r ."
```

Comment:
Note sure where this runs, but if it runs within the directory we package, we should delete the unzipped stuff.

Reply:
Build commands run (on MacOS) in the omnibus source directory, which is `/var/cache/omnibus/src/<software>`, so what happens inside this directory should not affect the final package.